backends/tfhe-cuda-backend/cuda/src/linearalgebra/addition.cuh

```diff
@@ -31,20 +31,20 @@ template <typename T>
 __host__ void
 host_addition_plaintext(cudaStream_t stream, uint32_t gpu_index, T *output,
                         T *lwe_input, T *plaintext_input,
-                        uint32_t lwe_dimension, uint32_t lwe_ciphertext_count) {
-
+                        uint32_t input_lwe_dimension, uint32_t input_lwe_ciphertext_count) {
   cudaSetDevice(gpu_index);
   int num_blocks = 0, num_threads = 0;
-  int num_entries = lwe_ciphertext_count;
+  int num_entries = input_lwe_ciphertext_count;
   getNumBlocksAndThreads(num_entries, 512, num_blocks, num_threads);
   dim3 grid(num_blocks, 1, 1);
   dim3 thds(num_threads, 1, 1);
 
-  cuda_memcpy_async_gpu_to_gpu(output, lwe_input,
-                               (lwe_dimension + 1) * lwe_ciphertext_count,
```

Comment:
Oh wow, the only change needed is to use
```
(lwe_dimension + 1) * lwe_ciphertext_count * sizeof(T)
```
in the call to `cuda_memcpy_async_gpu_to_gpu`